<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doctorWindow</class>
 <widget class="QWidget" name="doctorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>doctorWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Diagnosis:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="diagnosisLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Specialization:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="specializationLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Current doctor:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="currentDoctorLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Admission date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="dateLine"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="patientsCheckBox">
     <property name="text">
      <string>Only patients treaded by this doctor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="viewButton">
     <property name="text">
      <string>View</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="patientList"/>
   </item>
   <item>
    <widget class="QPushButton" name="addPatientButton">
     <property name="text">
      <string>Add new patient</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
